astropy/io/fits/hdu/compressed/header.py

```diff
@@ -765,11 +588,11 @@ def _image_header_to_bintable_header_and_coldefs(
     # that may be left over from the previous data
```

Comment:
This again should never be needed with a new `bintable`

Reply:
Fixed